src/test/java/net/masterthought/cucumber/generators/integrations/FeatureReportPageIntegrationTest.java

```diff
@@ -332,6 +332,34 @@ public void generatePage_OnRubyFormat_ForAfterHook_generatesEmbedding() {
         embeddingsElement[0].hasImageContent(embeddings[0]);
         asserEmbeddingFileExist(embeddings[0]);
     }
+    
+    @Test
+    public void generatePage_ForHook_generatesOutput() {
```

Comment:
as this feature is required for Ruby (and probably only for Ruby) refer to the naming convention of above method and apply